package.json

```diff
@@ -10,21 +10,22 @@
   "repository": "https://github.com/UPBGE/upbge.github.io",
   "dependencies": {
     "@heroicons/react": "^1.0.5",
+    "@popperjs/core": "^2.10.2",
     "html-to-react": "^1.4.7",
     "npm-check-updates": "^11.8.5",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
+    "react-device-detect": "^2.1.2",
```

Comment:
Do we actually need this? I don't see components or hooks from that module in files changed. Could you let me know where it's used?

Reply:
This commit was done during a test phase for Download page. I remove the commit now and I will add again when I do the proposal for Download.